examples/howto/hypothesis_testing.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<p>Useful to point out that there are an infinite number of intervals of a given probability, and that we are interested in the narrowest one.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/751/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>